app/move/app/new/controllers/save.js

```diff
@@ -32,6 +40,10 @@ class SaveController extends CreateBaseController {
         'documents',
       ])
       const move = await req.services.move.create(data)
+
+      // set before promises so it can be used for error handling
+      req.sessionModel.set('move', move)
```

Comment:
Just looking at the order, setting it after the Promise.all meant that it wouldn't be set if one of the promises errors out. Moving it here, it will be set beforehand. I don't know if that makes a different, the catch block will still fire and more to the error middleware.

Reply:
So I added a couple extra unit tests to capture it but moving it here means that if one of the update promises fails, like court hearings or profile update it will set this value and we can use it in the `errorHandler` of this controller.

It essentially means that if one of those fails we can trace the move that was created for them and any IDs that were in the session.